docs/products/postgresql/reference/pg-long-running-queries.rst

```diff
@@ -0,0 +1,81 @@
+PostgreSQL® long-running queries
+===============================
+
+Aiven does not terminate any customer queries even if they run indefinitely.  Long-running queries often cause issues by locking resources and preventing database maintenance tasks.
+
+To identify and terminate such long-running queries, you can do it from either:
+
+* `Aiven console <http://console.aiven.io>`_ / web page
+* `PostgreSQL® shell <https://www.postgresql.org/docs/current/app-psql.html>`_ (``psql``)
+
+
+Aiven console
+^^^^^^^^^^^^^
+In the Aiven web console, you can go to the **Current Queries** tab for your service.
```

Comment:
We usually refer to our console as the `Aiven Console` (check capitals) and always add the link to http://console.aiven.io

Reply:
added changes